concepts/eventing-code-guidelines.md

```diff
@@ -389,6 +389,64 @@ MarkReady(apiRuleNew) // instead consider using another function that explicitly
 ```
 </details>
 
+### Logging Guidelines
+
+Here are some general rules to standardize logs across eventing components:
+
+- Log message should be as short and meaningful as possible.
+- Logs should be aligned with the unified way of logging inside kyma ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/README.md)).
+- Each log message should have enough context to convey what happened.
+- Each log message should have the proper log level ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/unified-approach-to-logging-levels.md)).
+
+Code-specific rules:
+
+- Consider using `uber-go/zap` logging library. Label the logger by naming it with the component name and add the context whenever it is possible ([reference](https://github.com/kyma-project/community/blob/main/concepts/observability-consistent-logging/improvement-of-log-messages-usability.md#log-structure)):
+  <details>
+        <summary>Example</summary>
+
+  ```go
+  namedLogger := r.logger.WithContext().Named("logger-name").With("backend", "BEB")
+  r.namedLogger().Info("Creating Event Publisher deployment")
+  ```
+  will output:
+  ```
+  {"level":"INFO","timestamp":"2022-07-04T12:00:36+02:00","logger":"logger-name","caller":"backend/reconciler.go:741","message":"Creating Event Publisher deployment","context":{"backend":"BEB"}}
+  ```
+  </details>
+
+
+- Don't log the error in case you return the same error as a result for the `Reconcile()` method. **Reason:** kubebuilder will output it too, so in the end the user will have two very similar logs one after another:
+  <details>
+      <summary>Example</summary>
+
+    ```go
+    namedLogger.Errorw("Failed to sync BEB subscription", "error", err)
+    updateErr := r.updateSubscription(ctx, subscription, log); updateErr != nil {
+          return ctrl.Result{}, errors.Wrap(err, updateErr.Error())
+    }
+    ```
+  will result into duplication of logs:
+    ```
+    {"level":"ERROR","timestamp":"2022-07-01T08:20:26Z","logger":"beb-subscription-reconciler","caller":"beb/reconciler.go:275","message":"Failed to sync BEB subscription","context":{"kind":"Subscription","version":2,"namespace":"tunas-testing","name":"test-noapp","error":"prefix not found"}}
+    {"level":"ERROR","timestamp":"2022-07-01T08:20:26Z","caller":"controller/controller.go:326","message":"Reconciler error","context":{"controller":"beb-subscription-reconciler","object":{"name":"test-noapp","namespace":"tunas-testing"},"namespace":"tunas-testing","name":"test-noapp","reconcileID":"9994dd3e-0104-4170-82aa-79df9ec41af1","error":"prefix not found"}}
+    ```
+  </details>
+
+
+- Capitalize the component names, i.e. Event Publisher, or EventingBackend:
```

Comment:
```suggestion
- Capitalize the component names, for example, Event Publisher, or Eventing Backend:
```
[We don't like CamelCase](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#capitalized-terms). Is it necessary here?

Reply:
The problem is, that we already had this type of writing all over our eventing-controller component. And I think the link you mentioned is more applicable to docs, rather than to logs. In this document we talking purely about log output.

Reply:
Alright, the justification is sound :)

Reply:
Still, get rid of the abbreviation "i.e.".
If you want to keep the meaning, for the sake of simple language, I would usually suggest English "that is" instead of Latin "id est". 
But I think in this case "for example" might match better - those are examples, not an exhaustive list, right?

Reply:
The i.e. is already taken care of :)